libraries/stdlib/src/kotlin/random/Random.kt

```diff
@@ -267,10 +268,14 @@ public abstract class Random {
      *
      * @sample samples.random.Randoms.defaultRandom
      */
-    companion object Default : Random() {
+    companion object Default : Random(), Serializable {
+        private const val serialVersionUID = 0L
 
+        @Transient
```

Comment:
Ok, I think the variant with all transient fields and `readResolve` will do (without introducing a serialization surrogate).